ui/page.go

```diff
@@ -21,16 +21,16 @@ import (
 
 type pageIcons struct {
 	contentAdd, navigationCheck, actionCheckCircle, actionInfo, navigationArrowBack,
-	navigationArrowForward, actionCheck, chevronRight, actionSwapVert, navigationCancel,
-	imageBrightness1 *widget.Icon
+	navigationArrowForward, actionCheck, chevronRight, navigationCancel, navMoreIcon,
+	imageBrightness1, contentClear *widget.Icon
 
 	overviewIcon, overviewIconInactive, walletIconInactive, receiveIcon,
 	transactionIcon, transactionIconInactive, sendIcon, moreIcon, moreIconInactive,
 	pendingIcon, logo, redirectIcon, confirmIcon, newWalletIcon, walletAlertIcon,
 	importedAccountIcon, accountIcon, editIcon, expandIcon, copyIcon, mixer,
 	arrowForwardIcon, transactionFingerPrintIcon, settingsIcon, securityIcon, helpIcon,
 	aboutIcon, debugIcon, verifyMessageIcon, locationPinIcon, alertGray, arrowDownIcon,
-	watchOnlyWalletIcon *widget.Image
+	watchOnlyWalletIcon, currencySwapIcon, chevronDown, infoIcon, spinnerIcon *widget.Image
```

Comment:
* chevronDown is available in iconvg as ArrowDropdown.
* The "spinnerIcon" should be a loading animation not a static icon. Animations are yet to be added, so you can remove it for now.

Reply:
> chevronDown is available in iconvg as ArrowDropdown. 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/27733432/111785147-85813a00-88bc-11eb-8120-4fae19b61848.png">

Reply:
NavigationArrowDropDown. Before using an image for an icon, check here https://material.io/resources/icons/?icon=account_circle&style=baseline. If the icon is here, chances are that it exists in iconvg format.